docker-compose.override.yml

```diff
@@ -0,0 +1,9 @@
+version: "3"
+
+services:
+  harvest-admin:
```

Comment:
We should probably use consistent naming in order not to cause confusion, e.g. fdk-harvest-admin

Reply:
ok